vendor/k8s.io/kubernetes/plugin/pkg/admission/alwayspullimages/admission.go

```diff
@@ -53,7 +55,11 @@ func (a *alwaysPullImages) Admit(attributes admission.Attributes) (err error) {
 	if !ok {
 		return apierrors.NewBadRequest("Resource was marked with kind Pod but was unable to be converted")
 	}
-
+	// already scheduled pods cannot be mutated
+	if attributes.GetOperation() == admission.Update && len(pod.Spec.NodeName) != 0 {
```

Comment:
I don't know why this plugin is looking at update at all... ImagePullPolicy is never mutable